graphql_compiler/macros/macro_edge/helpers.py

```diff
@@ -88,7 +99,19 @@ def _replace_tag_names_in_tag_directive(name_change_map, tag_directive):
 
 
 def _replace_tag_names_in_filter_directive(name_change_map, filter_directive, ast):
-    """Return the new directive, and whether it is different from the old."""
+    """Return the new directive, and whether it is different from the old.
+
+    Args:
+        name_change_map: Dict[str, str] mapping tag names to new names
+        tag_directive: GraphQL library filter directive that potentially uses tagged parameters.
+                       all such tagged parameters should be in the name_change_map.
```

Comment:
Copy pasta. `tag_directive` is not an argument, and `filter_directive` and `ast` are not described.